config/environment.rb

```diff
@@ -4,7 +4,5 @@
 require File.expand_path('../application', __FILE__)
 
 # Initialize the rails application
-MyLibraryNYC::Application.initialize!
 
-Rails.logger = Logger.new(STDOUT)
```

Comment:
We already had ActiveSupport::Logger.new code  in environment(development.rb, qa.rb, production.rb) files. So here its not required this code.